packages/ui-components/src/Input/TextTypeInput.tsx

```diff
@@ -95,18 +88,9 @@ export const BaseTextInput: React.FC<InternalTextProps> = props => {
     </>
   );
 
-  const labelElement = existingPassword ? (
-    <div className="existing-password-label">
-      {labelDiv}
-      {forgotPasswordLinkElement}
-    </div>
-  ) : (
-    labelDiv
-  );
-
```

Comment:
no longer needed since we don't handle the forgot password link as part of the Input, and instead handle it in CC/DB Console.

Reply:
Nevermind, as per design in [DS-13](https://cockroachlabs.atlassian.net/browse/DS-13) forgot password should be displayed within Field